internal/pkg/manifest/lb_web_svc.go

```diff
@@ -202,3 +212,23 @@ func (s LoadBalancedWebService) ApplyEnv(envName string) (WorkloadManifest, erro
 	s.Environments = nil
 	return &s, nil
 }
+
+// WindowsCompatibility disallows unsupported services when deploying Windows containers on Fargate.
+func (s LoadBalancedWebService) WindowsCompatibility() error {
+	if !s.isWindows() {
+		return nil
+	}
+	// Exec is not supported.
+	if aws.BoolValue(s.ExecuteCommand.Enable) {
+		return errors.New(`'exec' is not supported when deploying a Windows container`)
+	}
+	// EFS is not supported.
+	if s.Storage != nil {
+		for _, volume := range s.Storage.Volumes {
+			if !volume.EmptyVolume() {
+				return errors.New(`'EFS is not supported when deploying a Windows container'`)
```

Comment:
nit: (the quotes seem to be in the wrong place)
```suggestion
				return errors.New(`'efs' is not supported when deploying a Windows container`)
```